src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator starts working");

```

Comment:
The message 'Excavator starts working' should be more informative and indicate that it is an Excavator that is working. Consider including the type of machine in the message.